components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-/* eslint-disable react/no-string-refs */
+
 /* eslint-disable max-lines */
```

Comment:
Disable max-lines is not removed here, instead the no-string-refs is. Is this on purpose?

Reply:
- this file slightly increase 800 lines, hence the max lines is not removed
- i have removed string ref eslint in favor of disabling on the particular line https://github.com/mattermost/mattermost-webapp/pull/10915/files/30447c0c9c3097c3c988f5e28102db50d1c5b044#diff-477d165c5acf38f82ef70bbea074b14877f86cf3669e4eabf6dbda6adffe086aR783 as inline forces us to see where is have disabled the lint in a file.